feat : added square and diamond shapes

mohittk opened the pull request:
I have added two shapes and working on adding more. 
![image](https://user-images.githubusercontent.com/79041510/153696729-9bc46d64-8ea8-4ce7-947a-a7db8202a249.png)

ashutosh1919 commented:
Test comment for discord bot to enable PR comments.

Sachin-chaurasiya commented:
@mohittk , I can see `yarn.lock` file in your PR. we are using npm for development. can you please make sure to use npm?

mohittk commented:
> @mohittk , I can see `yarn.lock` file in your PR. we are using npm for development. can you please make sure to use npm?

Ohh sorry for that! Yes will go for npm

Sachin-chaurasiya commented:
> > @mohittk , I can see `yarn.lock` file in your PR. we are using npm for development. can you please make sure to use npm?
> 
> Ohh sorry for that! Yes will go for npm

Also do not update the package if not required.

Sachin-chaurasiya commented on package.json:
```diff
@@ -21,7 +21,7 @@
     "react-tooltip": "^4.2.21",
     "theme-ui": "^0.13.1",
-    "typescript": "^4.5.4",
+    "typescript": "^4.5.5",
```
Please, do not update the package if not required.

mohittk commented:
> > > @mohittk , I can see `yarn.lock` file in your PR. we are using npm for development. can you please make sure to use npm?
> > 
> > 
> > Ohh sorry for that! Yes will go for npm
> 
> Also do not update the package if not required.

Okay sure! On it. Actually I was having some errors yesterday, so updated it. But I'll make sure for next PR.

Sachin-chaurasiya commented:
> > > > @mohittk , I can see `yarn.lock` file in your PR. we are using npm for development. can you please make sure to use npm?
> > > 
> > > 
> > > Ohh sorry for that! Yes will go for npm
> > 
> > 
> > Also do not update the package if not required.
> 
> Okay sure! On it. Actually I was having some errors yesterday, so updated it. But I'll make sure for next PR.

Can you revert the changes for the same?

mohittk commented:
> > > > > @mohittk , I can see `yarn.lock` file in your PR. we are using npm for development. can you please make sure to use npm?
> > > > 
> > > > 
> > > > Ohh sorry for that! Yes will go for npm
> > > 
> > > 
> > > Also do not update the package if not required.
> > 
> > 
> > Okay sure! On it. Actually I was having some errors yesterday, so updated it. But I'll make sure for next PR.
> 
> Can you revert the changes for the same?

You mean only to update package know?
Or anything else?

Sachin-chaurasiya commented:
@mohittk , can you also revert the [package-lock.json](https://github.com/ashutosh1919/react-awesome-shapes/pull/7/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) changes?

mohittk commented:
> @mohittk , can you also revert the [package-lock.json](https://github.com/ashutosh1919/react-awesome-shapes/pull/7/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) changes?
okay sure

mohittk commented:
@Sachin-chaurasiya done! check it once again. if any?

Sachin-chaurasiya commented:
> @Sachin-chaurasiya done! check it once again. if any?

@mohittk , I can still see the [package-lock.json](https://github.com/ashutosh1919/react-awesome-shapes/pull/7/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) file.

Sachin-chaurasiya commented:
@mohittk , I was asking for the removal of `package-lock.json` changes from your PR. You have removed the `package-lock.json` from the project itself.

mohittk commented:
> @mohittk , I was asking for the removal of `package-lock.json` changes from your PR. You have removed the `package-lock.json` from the project itself.

I already deleted my changes in last commit. You told that remove the package.json file. So I removed it. Please clarify in discord.

Sachin-chaurasiya commented:
@mohittk , your npm version is different, after installing the dependencies `package-lock.json` got updated with new lockfileversion, so I was asking you to do not to push those changes.

ashutosh1919 commented:
@Sachin-chaurasiya , @mohittk - this is good progress guys. Thanks for discussing. Please don't get disheartened with the errors. Also, note that whenever you commit files to raise PR, do not add `package-lock.json` file in the commit. It means that `git add <FILE_NAMES_SPACE_SEPARATED>` will not contain `package-lock.json`.

Sachin-chaurasiya commented:
@ashutosh1919 , we had a discussion and @mohittk has raised Separate PR.

mohittk commented:
> @Sachin-chaurasiya , @mohittk - this is good progress guys. Thanks for discussing. Please don't get disheartened with the errors. Also, note that whenever you commit files to raise PR, do not add `package-lock.json` file in the commit. It means that `git add <FILE_NAMES_SPACE_SEPARATED>` will not contain `package-lock.json`.

Yes! me along with @Sachin-chaurasiya solved the issue, and I created a separate PR. which is this one #7
